swift/ql/test/extractor-tests/generated/File/File.ql

```diff
@@ -0,0 +1,10 @@
+// generated by codegen/codegen.py
+import codeql.swift.elements
+import TestUtils
+
+from File x, string isUnknown, string getName
+where
+  toBeTested(x) and
+  (if x.isUnknown() then isUnknown = "yes" else isUnknown = "no") and
+  getName = x.getName()
+select x, "isUnknown:", isUnknown, "getName:", getName
```

Comment:
Should we maybe leave out the `"isUnknown:"` column if it's always emitted? It would reduce the noise a tiny bit in the expected output. It's probably good to have a name for the other columns (like `getName`), though.

Reply:
I don't know, we could accidentally remove extraction of an entity (maybe some stateless token without fields) and we would not see that. But now that I think about that, we can maybe add `not x.isUnknown()` to the `where` clauses and we would detect that as well

Reply:
Yeah, I think adding `not x.isUnknown()` would be a good idea.

Reply:
I've just committed that